src/lib/helpers.ts

```diff
@@ -245,21 +244,18 @@ function dec2hex(dec: number) {
 export function generatePKCEVerifier() {
   const verifierLength = 56
   const array = new Uint32Array(verifierLength)
-  if (!isBrowser()) {
-    for (let i = 0; i < verifierLength; i++) {
-      array[i] = Math.floor(Math.random() * 256)
-    }
-  } else {
-    window.crypto.getRandomValues(array)
+  if (typeof window.crypto === 'undefined') {
+    throw 'PKCE is not supported on devices without WebCrypto API support or equivalent polyfills'
```

Comment:
wouldn't this need to be `throw new Error('PKCE is not supported on devices without WebCrypto API support or equivalent polyfills')`?